book/plutocracy_report.ipynb

```diff
@@ -909,86 +302,29 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "One highly contentious proposal whose outcome did not change after filtering out whales was [this temperature check to choose which Eth <> BNB bridge to use for Uniswap v3 governance](https://snapshot.org/#/uniswap/proposal/0x6b8df360fdf73085b21fdf5eef9f85916fbde95621a3d454cb20fbe545ffc852). We see that even for the least popular choices, whales still contributed to the majority of the votes. We also observe that whales held the overwhelming majority of voting power for the runner-up, \"LayerZero\", with 99.9% of voting power allocated to this proposal being directed to this choice. By comparison, \"Wormhole\" received 65.1% of voting power for this proposal coming from whales."
+    "One highly contentious proposal whose outcome does not change after filtering out whales was [this temperature check to choose which Eth <> BNB bridge to use for Uniswap v3 governance](https://snapshot.org/#/uniswap/proposal/0x6b8df360fdf73085b21fdf5eef9f85916fbde95621a3d454cb20fbe545ffc852). Even for the least popular choices, whales contribute the majority of votes. We also observe that whales held the overwhelming majority of voting power for the runner-up, \"LayerZero\", with 99.9% of voting power allocated to the proposal behind it. By comparison, \"Wormhole\" received 65.1% of voting power for this proposal coming from whales."
```

Comment:
I'm still unsure how LayerZero could receive 99.9% of voting power while Wormhole received 65.1%. Can you clarify what you mean here?

Reply:
99.9% is actually the proportion of whale vp for **the choice**. Which isn't that useful. I'll change both to be the proportion of whale vp that chose those bridges out of the total vp of the proposal.